reana_cluster/backends/kubernetes/templates/deployments/workflow-controller-template.yaml

```diff
@@ -33,6 +33,18 @@ spec:
           value: "{{value}}"
         {% endfor %}
         {% endfor %}
+      - name: job-status-consumer
+        image: {{WORKFLOW_CONTROLLER_IMAGE}}
+        command: ["flask", "consume-job-queue"]
```

Comment:
This will override the Dockerfile command right?

Reply:
yes :)